### PR TITLE
Weekly cleanups

### PR DESCRIPTION
**Pull request by cdecker**
Just the usual bunch of drive-by fixes, mostly while trying to get Cirrus-CI to pass.

Changelog-None

### CODE REVIEW

File: contrib/pyln-testing/pyln/testing/utils.py [hunk 1]
```diff
@@ -1038,10 +1040,59 @@ def passes_filters(hmsg, filters):
...
+        with self.lock.acquire(poll_intervall=0.250):
+            # We just got the lock, assume someone else just released it
+            self.current_load = 100
+            while self.load() >= self.target:
+                time.sleep(1)
+
+            delay = time.time() - start_time
+            with open("/tmp/ltest-throttler.csv", "a") as f:
+                f.write("{}, {}, {}, {}\n".format(time.time(), self.load(), self.target, delay))
+            self.current_load = 100  # Back off slightly to avoid triggering right away
```
Comment (darosior) [inline contrib/pyln-testing/pyln/testing/utils.py:1084]: Read this comment a few times, but i still don't get it :sweat_smile: . What would a `< 75` load would trigger ? And why setting it to `100` after getting each node would avoid the trigger ?

**Review by darosior — APPROVED** [review]
LGTM, just a question about the throttler

File: contrib/pyln-testing/pyln/testing/utils.py (hunk 1 above)
Comment (cdecker) [reply contrib/pyln-testing/pyln/testing/utils.py:1084]: The thing is that if we leave the `current_load` at 75 percent a followup call, e.g., another `lightningd` wanting to start, could end up seeing 75% and decide to immediately schedule its startup (maybe the first one hasn't had time to start up and push the load up yet), thus we might end up clearing the queue despite only the first one getting the all-clear. 

Setting this to 100% means the ewma smoothing needs a couple of rounds before allowing the next one to go through, avoiding the flooding scenario above.

Comment (darosior) [reply contrib/pyln-testing/pyln/testing/utils.py:1084]: Ok, i had it the other way around, thanks!

**Review by darosior — APPROVED** [review]
ACK 336e972

**Pull request merged**